plugin/sublime-text/requirements.txt

```diff
@@ -0,0 +1,13 @@
+appdirs==1.4.4
+astroid==2.5.6
+black==21.6b0
+click==8.0.1
+isort==5.9.1
+lazy-object-proxy==1.6.0
+mccabe==0.6.1
+mypy-extensions==0.4.3
+pathspec==0.8.1
+pylint==2.8.3
+regex==2021.4.4
+toml==0.10.2
+wrapt==1.12.1
```

Comment:
In this case, these `requirements.txt` were supposed to be sort of equivalent to the `devDependencies` of `package.json`. But on second thought, I will remove the `requirements.txt` because, in practice, the file is useless, as the only things it contains are `pylint`, `black`, and the dependencies of both.

Reply:
aa0fc797c7b0536647b56f9d984662ace5699cda